docs/components/camera/_index.md

```diff
@@ -57,7 +57,7 @@ For configuration information, click on the model name:
 | [`align_color_depth_homography`](align-color-depth-homography/) | Uses a homography matrix to align the color and depth images. |
 | [`join_pointclouds`](join-pointclouds/) | Combines the point clouds from multiple camera sources and projects them to be from the point of view of target_frame. |
 | [`transform`](transform/) | A pipeline for applying transformations to an input image source. |
-| [`ultrasonic`](ultrasonic/) | An ultrasonic distance sensor like the [HC-S204](https://www.sparkfun.com/products/15569). |
+| [`ultrasonic`](ultrasonic/) | [HC-S204](https://www.sparkfun.com/products/15569) ultrasonic distance sensor. |
```

Comment:
```suggestion
| [`ultrasonic`](ultrasonic/) | The [HC-S204](https://www.sparkfun.com/products/15569) ultrasonic distance sensor. |
```

Or `Supports the ...` or similar. Just to make this also a sentence, matching above.